libs/auth-api-lib/src/lib/delegations/dto/delegation-index.dto.ts

```diff
@@ -11,6 +11,11 @@ export class DelegationRecordDTO {
   @IsString()
   @ApiProperty()
   toNationalId!: string
+
+  @IsOptional()
+  @IsString()
+  @ApiProperty({ type: String, nullable: true })
+  subjectId?: string | null
```

Comment:
Is this ever optional on the delegation index record?
We are always going to make sure that this is also inserted with the record, by creating a new subject ID if is does not exists in the user identities.